includes/Traits/Post_Type.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Trait Post_Type
+ *
+ * @package   Google\Web_Stories\Traits
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Traits;
+
+use WP_Post_Type;
+use WP_REST_Controller;
+use WP_REST_Posts_Controller;
+
+/**
+ * Trait Post_Type
+ *
+ * @package Google\Web_Stories\Traits
+ */
+trait Post_Type {
```

Comment:
Nit: pros of making this a trait instead of just namespaced functions?

Reply:
With was a suite of helper methods. Makes sense to me to group it together in a trait.